src/analyses/apron/affineEqualityAnalysis.apron.ml

```diff
@@ -4,11 +4,18 @@
 
 open Analyses
 
+open ArrayVector
+open ArrayMatrix
+
+open SparseVector
+open ListMatrix
```

Comment:
Do these opens then not shadow each other? Especially the different flavors of `Vector` and `Matrix`?